docs/README.md

```diff
@@ -97,7 +81,7 @@ The resulting image can be found by listing the images:
 ```console
 foo@bar:~$ docker images
 REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
-<none>              <none>              f541543bcd7f        64 seconds ago      119MB
+logdna-agent-v2     dcd54a0             e471b3d8a409        22 seconds ago      135MB
 ```
 
 ## Configuration
```

Comment:
can you add a third column title? also the spacing can be changed so that the second column has more width

Reply:
Added more width, but didn't understand what you meant with the first ask